src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Model/DigitalGreenCertificate.swift

```diff
@@ -122,6 +163,33 @@ public struct TestEntry: Codable, Equatable {
     public let certificateIssuer: String
     public let uniqueCertificateIdentifier: String
 
+    static func fake(
+        diseaseOrAgentTargeted: String = "diseaseOrAgentTargeted",
+        typeOfTest: String = "typeOfTest",
+        testResult: String = "testResult",
+        naaTestName: String? = nil,
+        ratTestName: String? = nil,
+        dateTimeOfSampleCollection: String = "dateTimeOfSampleCollection",
+        dateTimeOfTestResult: String = "dateTimeOfTestResult",
```

Comment:
I think it could make sense to have some default valid Date or date and time strings in the fakes

Reply:
done